### PR TITLE
Add PyAV recipe (non-Windows for now).

### PR DESCRIPTION
**Pull request by danielballan**
cc @caspervdw

### CODE REVIEW

**Comment by caspervdw** [comment]
Nice! I will have a look at the windows build as soon as the ffmpeg recipe is working.

**Comment by scopatz** [comment]
Looks great! Thanks @danielballan

File: recipes/PyAV/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{% set name = "av" %}
```
Comment (jakirkham) [inline recipes/PyAV/meta.yaml]: The directory and package name don't match. So this has failed to convert. Should it be `av` or `pyav`?

Comment (jakirkham) [inline recipes/PyAV/meta.yaml]: Please add a PR to correct this.

Comment (scopatz) [inline recipes/PyAV/meta.yaml]: Sorry!  I didn't know that was something I needed to be looking for :(  @danielballan would you mind clarifying and fixing, please? Thanks!

Comment (jakirkham) [inline recipes/PyAV/meta.yaml]: This is another occurrence of issue ( https://github.com/conda-forge/staged-recipes/issues/1073 ).

Comment (jakirkham) [inline recipes/PyAV/meta.yaml]: Sorry didn't see your comment when I posted that.

No worries. To be honest, I don't think this should be something that reviewers look for. As noted in that issue, it should either be the linter raising this before merging or the directory name should be ignored in favor of the package name.

Comment (scopatz) [inline recipes/PyAV/meta.yaml]: Fair enough :)

Comment (danielballan) [inline recipes/PyAV/meta.yaml]: The name on pip and the actual Python package name is av. The project calls itself PyAV everywhere else. Which is preferred?

**Comment by caspervdw** [comment]
Other thing: pyav is compiled against a given ffmpeg version. This is not an issue now as there is only one version on conda forge,but it will be in the future. Pinning ffmpeg seems to be the easiest option.

File: recipes/PyAV/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/PyAV/meta.yaml]: IMHO go for the Python package name (i.e. `av`). Many often assume `conda` is like `pip`. So it is good if we can match `pip`'s behavior.

**Comment by jakirkham** [comment]
Yep, it should be [pinned]( https://github.com/conda-forge/conda-forge.github.io/blob/cd299e04b0d4eba886326538177847b352fa2291/scripts/pin_the_slow_way.py#L31 ). Please update accordingly in the renaming PR.

File: recipes/PyAV/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/PyAV/meta.yaml]: Any chance you will have time to do this today, @danielballan? Currently this naming issue is causing us problems converting other recipes to feedstocks. If you don't have time, I think we should remove it and would then need you to re-add the recipe fresh in a new PR. Thoughts?

**Pull request merged**